upgrading/topics/rhsso/changes-72.adoc

```diff
@@ -31,4 +31,16 @@ Microsoft JDBC Driver 6.0 requires additional dependency added to the JDBC drive
 [source,xml]
 ----
 <module name="javax.xml.bind.api"/>
-----
\ No newline at end of file
+----
+
+=== Added session_state parameter to OpenID Connect Authentication Response
+
+OpenID Connect Session Management specification requires that parameter `session_state` is present in the OpenID Connect Authentication Response.
+In the RH-SSO 7.1, we didn't add this parameter, but now {project_name} 7.2 server adds this parameter by default as required per the specification.
+
+Problem is, that some OpenID Connect / OAuth2 adapters and especially older {project_name} adapters (RH-SSO 7.1 and older) may have issues with this new parameter.
+For example parameter will be always present in the browser URL after successful authentication to the client application. So if you use RH-SSO 7.1 or
+some other legacy OAuth2 / OpenID Connect adapter, it may be useful to disable adding session_state parameter to the authentication response.
+It can be done for the particular client in {project_name} admin console in client details and section with `OpenID Connect Compatibility Modes`
+described in <<_compatibility_with_older_adapters>> . There is the switch `Exclude Session State From Authentication Response`, which can be
+turned on to not add the `session_state` parameter to the Authentication Response.
```

Comment:
The OpenID Connect Session Management specification requires that the parameter `session_state` is present in the OpenID Connect Authentication Response.

In RH-SSO 7.1, we did not have this parameter, but now {project_name} adds this parameter by default, as required by the specification.

However, some OpenID Connect / OAuth2 adapters, and especially older {project_name} adapters (such as RH-SSO 7.1 and older), may have issues with this new parameter.

For example, the parameter will be always present in the browser URL after successful authentication to the client application. If you use RH-SSO 7.1 or a legacy OAuth2 / OpenID Connect adapter, it may be useful to disable adding the `session_state` parameter to the authentication response. This can be done for the particular client in the {project_name} admin console, in client details in the section with `OpenID Connect Compatibility Modes`, described in <<_compatibility_with_older_adapters>>. There is the `Exclude Session State From Authentication Response` switch, which can be turned on to prevent adding the `session_state` parameter to the Authentication Response.